io.github.Fontforge.appdata.xml

```diff
@@ -0,0 +1,98 @@
+  створювати, редагувати та перетворювати шрифти у форматах PostScript, TrueType, OpenType,
+  cid-keyed, multi-master, cff, SVG та растрових (bdf, FON, NFNT).</p>
+  <p xml:lang="pt">
+  FontForge — um editor de fontes para fontes outline e de bitmap que lhe permite criar,
+  editar ou converter, uma série de fontes, incluindo PostScript, TrueType, OpenType,
+  cid-keyed, multi-master, cff, SVG e de bitmap (bdf, FON, NFNT) fontes.</p>
+  <p xml:lang="pt">
+  Aprender a usar FontForge é fácil, e existem vários tutoriais disponíveis que começam
+  com o básico até mais recursos avançados, como fazer e usar scripts.</p>
+  <p xml:lang="zh_CN">
+  FontForge是一个矢量字体和位图字体的编辑器。你可以用它来创造，编辑或者转换一些字体。支持的字体格式包括PostScript，TrueType，OpenType，CID-keyed，multi-master，CFF，SVG和位图（bdf，PON，NFNT）。</p>
+  <p xml:lang="zh_CN">
+  FontForge是一个自由开源软件，可以运行在多种计算机操作系统上。你可以使用FontForge的图形界面，也可以将其作为一个命令行工具来使用。</p>
+  <p xml:lang="zh_CN">
+  学习使用FontForge并不难，有各种各样的教程涵盖了基础知识到更高级的特性，比如制作和使用脚本。</p>
+  <p xml:lang="ko">
+  폰트포지는 외곽선 및 비트맵 글꼴 편집기입니다. 포스트스크립트, 트루타입, 오픈타입, CID 폰트,
+  멀티마스터, CFF, SVG 그리고 비트맵(bdf, FON, NFNT) 글꼴을 제작, 편집, 변환할 수 있습니다.</p>
+  <p xml:lang="ko">
+  폰트포지는 여러 컴퓨터 운영체제에서 작동하도록 쓰여진 오픈소스 소프트웨어입니다.
+  명령어 인터페이스 또는 GUI로 사용할 수 있습니다.</p>
+  <p xml:lang="ko">
+  폰트포지는 배우기 쉽습니다. 기본적인 것부터 시작해서, 스크립트를 짜서 쓰는 고급 기능까지
+  다양한 튜토리얼들이 준비되어 있습니다.</p>
+ <p xml:lang="ca">
+  El FontForge és un editor de tipus de lletra, tant vectorials com de mapa de bits,
+  que permet crear, editar o convertir tipus de diferents formats, com ara PostScript,
+  TrueType, OpenType, CID, MM, CFF, SVG i BitMap (BDF, FON, NFNT).</p>
+ <p xml:lang="ca">
+  El FontForge és programari lliure dissenyat per poder ser utilitzat en diversos
+  sistemes operatius, tant des de la interfície gràfica com des de la línia d'ordres.</p>
+ <p xml:lang="ca">
+  Aprendre a utilitzar el FontForge és senzill; podeu consultar els tutorials
+  existents que van de les funcions més bàsiques del programa fins a les més complicades,
+  com ara escriure i usar scripts.</p>
+  <p xml:lang="it">
+  FontForge è un editor di font per i caratteri di testo outline e bitmap che ti permette di creare,
+  modificare o convertire diversi tipi di font, inclusi PostScript, TrueType, OpenType,
+  CID, multi-master, cff, SVG e BitMap (bdf, FON, NFNT).</p>
+  <p xml:lang="it">
+  FontForge è un Software Open Source gratuito, scritto per funzionare su vari sistemi
+  operativi. Puoi usare FontForge con interfaccia grafica o tramite riga di comando.</p>
+  <p xml:lang="it">
+  Imparare ad usare FontForge è facile, e sono disponibili vari tutorial che iniziano dalle basi
+  fino ad arrivare alle funzioni avanzate, come la creazione e l'uso di script.</p>
+ </description>
+ <screenshots>
+  <screenshot type="default" width="1200" height="675">https://github.com/fontforge/fontforge/raw/master/doc/html/ff-screenshot.png</screenshot>
+ </screenshots>
+  <url type="homepage">https://github.com/fontforge/fontforge</url>
```

Comment:
You could at least add url for issues & maybe donation.

Reply:
good hint. done.